File: AppGenerateFiles/dbo/FrontModel/Transaction_Movimiento.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
class Transaction_Movimiento extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_movimiento;
   /**@type {String}*/ descripcion;
   /**@type {String}*/ concepto;
   /**@type {Number}*/ id_usuario_crea;
   /**@type {Date}*/ fecha;
   /**@type {String}*/ tipo;
   /**@type {String}*/ moneda;
   /**@type {Number}*/ tasa_cambio;
   /**@type {Number}*/ tasa_cambio_compra;
   /**@type {Boolean}*/ correo_enviado;
   /**@type {Boolean}*/ is_transaction;
}
export { Transaction_Movimiento }

File: AppGenerateFiles/dbo/Model/Transaction_Movimiento.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaction_Movimiento : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_movimiento { get; set; }
       public string? descripcion { get; set; }
       public string? concepto { get; set; }
       public int? id_usuario_crea { get; set; }
       public DateTime? fecha { get; set; }
       public string? tipo { get; set; }
       public string? moneda { get; set; }
       public Double? tasa_cambio { get; set; }
       public Double? tasa_cambio_compra { get; set; }
       public bool? correo_enviado { get; set; }
       public bool? is_transaction { get; set; }
   }
}
